strawberry/codegen/query_codegen.py

```diff
@@ -751,7 +755,13 @@ def _collect_types_using_fragments(
         for fragment in fragments:
             fragment_class_name = class_name + fragment.type_condition.name.value
 
-            current_type = GraphQLObjectType(fragment_class_name, list(common_fields))
+            typename = None
+            with contextlib.suppress(AttributeError):
+                typename = fragment.type_condition.name.value
```

Comment:
can we change this to use get attribute? I think it is a bit more explicit of what's happening :)

Reply:
It turns out that we don't need to worry about the `AttributeError` here -- we use the exact same `fragment.type_condition.name.value` a couple lines up (and then again later in the loop).  This makes sense ... I think that fragments _have_ to be `on` _something_.

I guess there's an [edge case](https://spec.graphql.org/October2021/#sec-Inline-Fragments) for inline fragments with `@include(if: ...)` directives but the current code doesn't handle those (and I'm not really sure how I would expect the codegen to handle an `@include(if: ...)` anyway...

In any case, I'll leave that handling for a future PR if that's OK with you 😅

Reply:
sure! thanks 😊